README.md

```diff
@@ -27,10 +31,18 @@ class Vehicle(vqpy.VObjBase):
         return self.infer('license_plate', {'license_plate': 'openalpr'})
 ```
 
+Here you can see a builtin interface `infer` provided by the library. There are two basic interfaces `infer` and `getv`, both of which are used to infer an attribute, typically an user-defined or pre-defined property in most cases. There are two differences in the functions: `getv` can retrieve values in historic frames if the property is marked by `@vqpy.stateful` decorator; `getv` consider user-defined names first and `infer` directly look up the library.
+
+We also have some other decorators for defining properties, including `@vqpy.postproc` and `@vqpy.cross_object_property`. The former decorator provides basic postprocessing functions. For instance, returning the modal value of the results in past 10 frames. The latter decorator can help user define cross-object properties, so that users can track values like distances between all pairs of people and baggage.
```

Comment:
I think this should be in the API document, because if we have many postprocessing functions, we do not need to introduce them separately in this README.

Reply:
That makes sense. Then we can remove "For instance, returning the modal value of the results in past 10 frames."

Reply:
Nitpick:  
1. "like @vqpy.postproc. This decorator provides..." -> "like @vqpy.postproc, which provides..." 
2. A separate paragraph for "You can also refer to our [demos] to see several example implementations of concerned objects."